features/share/graphql/node.js/mutation.cjs

```diff
@@ -0,0 +1,99 @@
+const { GraphQLClient, gql } = require('graphql-request');
```

Comment:
I'd change the containing directory from `node.js` to `node` or `nodejs` for ease of navigation.